packages/victory-axis/src/helper-methods.tsx

```diff
@@ -595,7 +595,8 @@ export const getBaseProps = (props, fallbackProps) => {
         ? { [otherAxis]: props.scale[otherAxis] }
         : undefined,
   };
-  return ticks.reduce((childProps, tickValue, index) => {
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+  return (ticks as number[]).reduce((childProps, tickValue, index) => {
```

Comment:
This seems weird, is this change necessary?

Reply:
It was giving a type check error without it. I assume since i force [''] into what is expecting a number[]. I'm sure there may be a cleaner way to do it.

Reply:
The reason it's weird, though, is because of the need to disable `no-unnecessary-type-assertion` ... which is basically saying "it's already `number[]`, no need for `as number[]`.
Can you check what happens if you revert this change?  Maybe it was broken during development but now it can be changed back?
If it's still broken, then it's ok to merge.

Reply:
<img width="1120" alt="Screenshot 2023-11-12 at 11 55 35 AM" src="https://github.com/FormidableLabs/victory/assets/755626/a8c05f24-c994-4807-bccc-3b02c44d1ef8">